src/main/java/uk/gov/hmcts/ccd/domain/types/DateValidator.java

```diff
@@ -43,7 +43,7 @@ public BaseType getType() {
         try {
             dateValue = LocalDate.parse(dataValue.asText(), ISO_DATE);
         } catch (DateTimeParseException e) {
-            return Collections.singletonList(new ValidationResult(dataValue + " is not a valid ISO 8601 date", dataFieldId));
+            return Collections.singletonList(new ValidationResult("Date or Time entered is not valid", dataFieldId));
```

Comment:
Better to create a constant for the error message "Date or Time entered is not valid", especially if it is used in multiple places